integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/datamap/CarbonCreateDataMapCommand.scala

```diff
@@ -106,7 +106,8 @@ case class CarbonCreateDataMapCommand(
         .exists(_.getDataMapName.equalsIgnoreCase(dataMapSchema.getDataMapName))) {
         if (!ifNotExistsSet) {
           throw new MalformedDataMapCommandException(
-            "DataMap with name " + dataMapSchema.getDataMapName + " already exists in storage")
+            "Materialized view with name " + dataMapSchema.getDataMapName +
```

Comment:
Since you have added a command to create mv, is this check in createdatamap is required?

Reply:
I will revert this change. This PR only modifies MV related command, DataMap and index related command modification will be done in #3614